internal/xds/translator/translator.go

```diff
@@ -93,6 +94,18 @@ func Translate(ir *ir.Xds) (*types.ResourceVersionTable, error) {
 		}
 
 		xdsRouteCfg.VirtualHosts = append(xdsRouteCfg.VirtualHosts, vHost)
+
+		// TODO: Make this into a generic interface for API Gateway features
```

Comment:
It would be nice if the TODO references a GH issue.

Reply:
sure will raise a GH issue once this PR is merged and link it

Reply:
raised https://github.com/envoyproxy/gateway/issues/882 to track this